contracts-periphery/src/WithdrawHook.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+pragma abicoder v2;
+
+import "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
+import "src/utils/ISwapRouter.sol";
+
+contract WithdrawHook {
```

Comment:
This file and contract should be named something more specific, like `WithdrawHookUniswap`. Otherwise we'll have to rename this once we add a second withdrawal hook contract for another protocol integration

Reply:
I would probably swap the order, and name it something like `UniswapWithdrawHook`